docs/registry.md

```diff
@@ -0,0 +1,134 @@
+ # Registry Compatibility with SOCI
+
+SOCI is compatible with most registries. To check if your registry of choice is compatible, see [List of Registry Compatibility](#list-of-registry-compatibility).
+
+For most use-cases, compatibility is the only concern. However, there is a difference in *how* registries work with SOCI that could cause surprising edge cases. The rest of this document is a technical dive into how SOCI indices are stored and retrieved from registries and the surprises you might encounter.
+
+## Registry Requirements
+
+In order for a registry to be compatible SOCI it must support the following features of the OCI distribution and image specs:
+
+1) Accept [OCI Image Manifests](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/manifest.md) with [subject fields](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/manifest.md#image-manifest-property-descriptions) and arbitrary config media types.
+This allows the registry to store SOCI indices and is supported by most registries.
+
+2) (optional) Support the [OCI referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#listing-referrers)
+This adds convenience around retrieving SOCI indices from the registry. If it is not supported, there is a [fallback mechanism](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#unavailable-referrers-api) that can be used instead, but it has a few issues noted in the next section.
```

Comment:
> there is a [fallback mechanism](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#unavailable-referrers-api) that can be used instead

It's unclear to me how this can be used. Is it something a registry needs to support? (i.e., do you mean "If it's not supported, the registry needs to support the fallback mechanism").